### PR TITLE
Card component improvement

### PR DESCRIPTION
**Pull request by avicndugu**
Add error handling in during when fetching data

### CODE REVIEW

File: src/RestaurantCardsContainer.jsx [hunk 1]
```diff
@@ -8,35 +8,57 @@ class RestaurantCardsContainer extends React.Component {
...
-      restaurants: []
+      restaurants: [],
+      error: null,
     };
   }
 
   componentDidMount() {
-    fetch('https://learnreact.avicndugu.repl.co/restaurants.json')
-      .then(res => res.json())
+    fetch('https://learnreact.avicndugu.repl.co/resstaurants.json')
```
Comment (themythia) [inline src/RestaurantCardsContainer.jsx]: There's a typo in the URL.  Is this to test error handling?

File: src/RestaurantCardsContainer.jsx [hunk 2]
```diff
@@ -8,35 +8,57 @@ class RestaurantCardsContainer extends React.Component {
...
+            {restaurants.map((restaurant, index) => (
+              <Col>
+                <p>{error}</p>
+                <RestaurantCard
+                  key={index}
+                  name={restaurant.restaurantName}
+                  address={restaurant.address}
+                  hours={restaurant.hours}
+                />
+              </Col>
```
Comment (themythia) [inline src/RestaurantCardsContainer.jsx]: `key={index}` should be passed to `<Col>` at line 48.

**Review by themythia — APPROVED** [review]

File: src/RestaurantCardsContainer.jsx (hunk 2 above)
Comment (avicndugu) [reply src/RestaurantCardsContainer.jsx]: I have moved ```key={index}``` to ```<Col>```

File: src/RestaurantCardsContainer.jsx (hunk 1 above)
Comment (avicndugu) [reply src/RestaurantCardsContainer.jsx]: That is definately a typo. I forgot to change it back to original url after testing.

I have corected that.

**Pull request merged**